config.tpl.yml

```diff
@@ -264,6 +264,13 @@ images:
   #   os_type: Linux # Linux or Windows
   #   version: 7.9 # Version of the image to create the image definition in SIG. Pattern is major.minor where minor is mandatory
 # Pre-defined images
+  - name: azhop-almalinux86-v2-rdma-gpgpu
```

Comment:
This name doesn't map to any packer ones if customer wants to use it. The provided one is to build marketplace image, not custom images. Change the name to the packer script file name.